### PR TITLE
Updated README docs for terraform modules

### PR DESCRIPTION
**Pull request by JoeCSykes**

### CODE REVIEW

File: Terraform_modules/ec2_instance/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+This allows bash scripts and command line commands to be specified and run in the EC2 instance when launched. Do not pass gzip-compressed data via this argument.
+
+**needs_elastic_ip (Optional)**
+
+This is a boolean value inidcating whether an elastic IP should be generated and associated with the EC2 instance. The default value is `false`.
+
+**user_data_replace_on_change (Optional)**
+
+This vaule indicates whether changes to the `user_data` value triggers a rebuild of the EC2 instance. The default value is `true`.
```
Comment (bethcryer) [inline Terraform_modules/ec2_instance/README.md:51]: *value

File: Terraform_modules/ec2_instance/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+This is the subnet id you want the ec2 instance to be created in.
+
+**vpc_security_group_ids (Mandatory)**
+
+This is a list of ids that specifies the security groups you want your EC2 to be in. If you do not wish to specify a security group for your module then please set 
+this value to an empty list.
+
+**ec2_instance_type (Optional)**
+
+This is the type of EC2 instane you want (e.g. "t2.micro", "m5.large" etc.). The default value for this is "t2.micro" as this is the free tier instance.
```
Comment (bethcryer) [inline Terraform_modules/ec2_instance/README.md:35]: *instance

File: Terraform_modules/ec2_instance/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+# Terraform EC2 instance Module
+
+This Terraform module will produce an EC2 instance which can be accessed via ssh or session manager via the aws management console. It also allows the posibility
```
Comment (bethcryer) [inline Terraform_modules/ec2_instance/README.md:3]: *possibility

File: Terraform_modules/rds_serverless_cluster/README.md [hunk 4]
```diff
@@ -1,68 +1,85 @@
 # Terraform RDS Serverless Module
 
-This Terraform module will produce a 
-
+This Terraform module will produce a Aurora serverless database inside a database cluster. The serverless database can be of two types `aurora-mysql` or `aurora-postgresql`. AWS Secrets manager is used to store the default username and password given to the serverless database where the password is a randomly generated sequence of 16 characters. These secrets can be viewed in the Secrets Manager section of the aws management console. These secrets are also set as outputs of the module and can be used therought your terraform code.
 
 # Inputs
-** project_name (Manadatory)
+**project_name (Manadatory)**
```
Comment (bethcryer) [inline Terraform_modules/rds_serverless_cluster/README.md:6]: *Mandatory

**Review by bethcryer — CHANGES_REQUESTED** [review]
only issue I saw were typos, otherwise the readme looks super comprehensive 👍

File: Terraform_modules/ec2_instance/README.md (hunk 1 above)
Comment (JoeCSykes) [reply Terraform_modules/ec2_instance/README.md]: Done

File: Terraform_modules/ec2_instance/README.md (hunk 2 above)
Comment (JoeCSykes) [reply Terraform_modules/ec2_instance/README.md]: Done

File: Terraform_modules/ec2_instance/README.md (hunk 3 above)
Comment (JoeCSykes) [reply Terraform_modules/ec2_instance/README.md]: Done

File: Terraform_modules/rds_serverless_cluster/README.md (hunk 4 above)
Comment (JoeCSykes) [reply Terraform_modules/rds_serverless_cluster/README.md]: Done

**Review by bethcryer — APPROVED** [review]

**Pull request merged**